---
title: Dopetrope
layout: default
---
	<body class="no-sidebar">

		{% include nav.html %}

		<!-- Main -->
			<div id="main-wrapper">
				<div class="container">

					<!-- Content -->
						<article class="box post">
							<a href="#" class="image featured"><img src="{{ site.baseurl }}/assets/images/pic01.jpg" alt="cover photo" /></a>
							<header>
								<h2>Research Projects</h2>
							</header>
                            <p>
                                    I study the connection between brain structure and function in the brain of mammals. Specifically, I analyze the affect of genes on various brain structures in humans and mice.
                            </p>
                            <p>
                                Also, I develop new machine learning algorithms and machine vision tools. These tools and algorithms can be applied to genomic data but are not limited to that use. 
                            </p>
							
							<section id="develop_origin">
                                <header>
									<h3><a href="http://dx.doi.org/10.1371/journal.pcbi.1005064" >On expression patterns and developmental origin of human brain regions</a></h3>
                                    <p></p>
								</header>
                                <div class="row">
                                    <div class="4u">
                                        <a href="http://dx.doi.org/10.1371/journal.pcbi.1005064" class="image featured">
                                            <img id="BRO_figure" alt="BRO - On expression patterns and developmental origin of human brain regions" src="assets/img/bro_figure1.png" /> 
                                        </a>
                                    </div>
                                    <div class="8u">
                                        <p>
                                            Genome-wide measurements of gene expression across the human brain can reveal new
                                            principles of brain organization and function. To achieve this, we aim to discover which 
                                            genes are differentially expressed and in what brain regions. We found that almost all 
                                            genes in the adult human brain bear a developmental ‘footprint’ which determines their 
                                            areal expression-pattern based on the developmental ontology of brain regions, while at 
                                            the same time their spatial expression pattern changes during life. Furthermore, pairs of 
                                            paralog genes with stronger footprint, tend to be more strongly correlated (or anti 
                                            correlated) suggesting that their expression is more strongly spatially tuned, and these 
                                            effects are stronger in paralog of synaptic pathways.
                                        </p>
                                        <p>
                                              <a href="http://dx.doi.org/10.1371/journal.pcbi.1005064" >"On Expression Patterns and Developmental Origin of Human Brain Regions" (PLoS Computational Biology, 2016)</a>.
                                        </p>
                                    </div>
                                </div>
                                
							</section>
                            
                            <section id="demixing">
								<header>
									<h3>Multi-region demixing of brain transcriptome</h3>
                                    <p></p>
								</header>
                                <div class="row">
                                    <div class="4u">
                                        <a href="#" class="image featured">
                                            <img id="cellmix_illustration" alt="cellmix illustration - seperating a mixture of celltypes" src="assets/img/cellmix.jpg" />
                                        </a>
                                    </div>
                                    <div class="8u">
                                        <p>
                                            High quality and large scale measurements of gene expression in the brain have been collected across many species and brain regions, but unfortunately, almost all these measurements aggregate over an uncontrolled mix of various cell types including neurons, glia and blood cells. This fact hinders the insights that can be gained by analyzing datasets of gene expression in the brain. Inferring cell-type specific information from these mixtures is a key challenge. 
                                        </p>
                                        <p>
                                            We propose here a probabilistic approach to blindly demix brain transcriptome samples, building on variability of cell-type proportions across samples. We further describe a model that takes into account that tissues in different brain structures differ in their expression profiles by introducing soft-sharing between cell-type-specific profiles from different brain regions. We describe an efficient algorithm to estimate the parameters of the model, as well as techniques to determine some of its hyper parameters from prior knowledge. We study the regimes where multi-region demixing works well using controlled data and evaluate the approach using RNA-seq profiles measured from multiple regions in postmortem human brains. Predictions are validated against ground-truth data of cell-type specific profiles measured in corresponding regions in the mouse brain, showing that soft demixing successfully outperforms baselines of demixing unified and individual regions.
                                        </p>
                                        <p>Under review</p>
                                    </div>
                                </div>
								
							</section>
                            
                            <section id="cerbellar">
								<header>
									<h3>
                                        <a href="http://dx.doi.org/10.1371/journal.pcbi.1002790" >Localizing Genes to Cerebellar Layers by Classifying ISH Images</a>
                                    </h3>
                                    <p></p>
								</header>
                                <div class="row">
                                    <div class="4u">
                                        <a href="http://dx.doi.org/10.1371/journal.pcbi.1002790" class="image featured">
                                            <img id="CB" src="assets/images/pic04.jpg" alt="gene markers in the cerebellum of mice"/>
                                            <!--<img id="CB" src="assets/img/cb.png" />-->
                                        </a>
                                    </div>
                                    <div class="8u">
                                        <p>
                                            The way gene expression is spatially distributed across the brain reflects the function and micro-structure of neural tissues. Measuring these patterns is hard because brain tissues are composed of many types of neurons and glia cells, and average gene expression across a region mixes transcripts from many different cells. We present here an approach to identify genes that are primarily expressed in specific brain layers or cell types, based on analyzing high resolution in-situ hybridization images. By learning the spatial patterns of a few known cell markers, we annotate the expression patterns of hundreds of new genes, and predict the layers and cell types they are expressed in.
                                        </p>
                                        <p>
                                              Here's our paper <a href="http://dx.doi.org/10.1371/journal.pcbi.1002790" >"Localizing genes to brain layers using ISH image classification" (PLoS Computational Biology, 2012)</a> and its <a href="http://chechiklab.biu.ac.il/~lior/cerebellum2.html">accompanying webpage</a>.
                                        </p>
                                    </div>
                                </div>
							</section>
                            
							<section id="net_opt">
								<header>
									<h3>
                                        <a href="http://dx.doi.org/10.1088/1367-2630/18/9/093039" >
                                        Optimal network structure for spectral radius related phase transitions
                                        </a>
                                    </h3>
                                    <p></p>
								</header>
                                <div class="row">
                                    <div class="4u">
                                        <a href="http://dx.doi.org/10.1088/1367-2630/18/9/093039" class="image featured">
                                            <img id="CB" src="assets/images/flight_net_b.jpg" alt="optimization of the world flight network"/>
                                        </a>
                                    </div>
                                    <div class="8u">
                                        <p>
                                            The dynamics of contact processes on networks is often determined by the spectral radius of the networks adjacency matrices. A decrease of the spectral radius can prevent the outbreak of an epidemic, or impact the synchronization among systems of coupled oscillators. The spectral radius is thus tightly linked to network dynamics and function. As such, finding the minimal change in network structure necessary to reach the intended spectral radius is important theoretically and practically. Given contemporary big data resources such as large scale communication or social networks, this problem should be solved with a low runtime complexity.
                                        </p>
                                        <p>
                                            We introduce a novel method for the minimal decrease in weights of edges required to reach a given spectral radius. The problem is formulated as a convex optimization problem, where a global optimum is guaranteed. The method can be easily adjusted to an efficient discrete removal of edges. We introduce a variant of the method which finds optimal decrease with a focus on weights of vertices. The proposed algorithm is exceptionally scalable, solving the problem for real networks of tens of millions of edges in a short time.
                                        </p>
                                        <p>
                                              <a href="http://dx.doi.org/10.1088/1367-2630/18/9/093039" >"Optimal network modification for spectral radius dependent phase transitions" (New Journal of Physics, 2016)</a>.
                                        </p>
                                    </div>
                                </div>
							</section>
						</article>

				</div>
			</div>
